src/components/question_freetext.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div id="textarea">
+    <div class="questions">
+      <p class="question_title"></p>
+      <div class="question_answer">
+        <textarea name="kanso" rows="15" cols="71"></textarea>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  data: function(){
+    return{
```

Comment:
`{}`の両外側にはスペースを空けるようにしましょう。
他の部分でも同様です。
```suggestion
    return {
```